thread '<unnamed>' panicked at lib/src/metta/types.rs:142:22:
Incorrect function type: (->)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at core/src/panicking.rs:221:5:
panic in a function that cannot unwind
stack backtrace:
   0:     0x7f204f62f0aa - <std::sys::backtrace::BacktraceLock::print::DisplayBacktrace as core::fmt::Display>::fmt::h304520fd6a30aa07
   1:     0x7f204f67c5eb - core::fmt::write::hf5713710ce10ff22
   2:     0x7f204f6236a3 - <unknown>
   3:     0x7f204f6318a2 - <unknown>
   4:     0x7f204f63150e - std::panicking::default_hook::h81c8cd2e7c59ee33
   5:     0x7f204f6321ef - std::panicking::rust_panic_with_hook::had2118629c312a4a
   6:     0x7f204f631ea3 - <unknown>
   7:     0x7f204f62f589 - <unknown>
   8:     0x7f204f631b64 - rust_begin_unwind
   9:     0x7f204f24a935 - core::panicking::panic_nounwind_fmt::hc0ae93930ea8f76c
  10:     0x7f204f24a9c2 - core::panicking::panic_nounwind::h9f485ff9b02bac75
  11:     0x7f204f24abe6 - core::panicking::panic_cannot_unwind::hea865182d7ce50af
  12:     0x7f204f28e962 - metta_run
  13:     0x7f204f26df7c - <unknown>
  14:     0x7f204f2553ef - <unknown>
  15:     0x5632e43a2282 - <unknown>
  16:     0x5632e4398b4b - _PyObject_MakeTpCall
  17:     0x5632e43924c8 - _PyEval_EvalFrameDefault
  18:     0x5632e43a2aec - _PyFunction_Vectorcall
  19:     0x5632e438cae8 - _PyEval_EvalFrameDefault
  20:     0x5632e43a2aec - _PyFunction_Vectorcall
  21:     0x5632e438c9a2 - _PyEval_EvalFrameDefault
  22:     0x5632e4471e56 - <unknown>
  23:     0x5632e4471d26 - PyEval_EvalCode
  24:     0x5632e4498ae8 - <unknown>
  25:     0x5632e44932ef - <unknown>
  26:     0x5632e4498885 - <unknown>
  27:     0x5632e4497e68 - _PyRun_SimpleFileObject
  28:     0x5632e4497b47 - _PyRun_AnyFileObject
  29:     0x5632e448c02e - Py_RunMain
  30:     0x5632e4465d6d - Py_BytesMain
  31:     0x7f20500f8d90 - <unknown>
  32:     0x7f20500f8e40 - __libc_start_main
  33:     0x5632e4465c65 - _start
  34:                0x0 - <unknown>
thread caused non-unwinding panic. aborting.
timeout: the monitored command dumped core
real 0.30
user 0.18
sys 0.00
